src/ledger/cleanup_service.zig

```diff
@@ -11,7 +11,7 @@ const Slot = sig.core.Slot;
 
 const BlockstoreDB = ledger.BlockstoreDB;
 const BlockstoreReader = ledger.reader.BlockstoreReader;
-const BlockstoreWriter = ledger.writer.BlockstoreWriter;
+const LedgerResultWriter = ledger.result_writer.LedgerResultWriter;
```

Comment:
Should `BlockStoreReader` be also updated to `LedgerResultReader`? To keep it symmetric.

Reply:
I made this change partly because the prior names indicated a false symmetry. I feel that it was misleading.

Writing data is split into two categories. Each category has a separate struct, named based on the category
- ShredInserter: insert shreds and related metadata
- LedgerResultWriter: save the results of executing blocks and transactions

All reads from the blockstore are handled by the BlockstoreReader. It doesn't matter if you want to read transactions or the results of those transactions. It's all one struct. BlockstoreReader is not specific to just "results"